### DISCUSSION TITLE
Feature request: Improved tag autocompletion

### DISCUSSION THREAD

**Discussion by fachinformatiker** [go-shiori/shiori#357] [Feature requests]
Hello
It would be nice to have a list of all tags to chose from or create a new one when you create a new bookmark.
I had the problem quite a lot, when I create bookmarks and use "tutorial" for example and later on "tutorials".
Now I have two tags and need to move bookmarks.
Thanks

**Comment by unoukujou** [comment]
Wow, a 2 & ½ year old bump. But I came here to request exactly this for the exact same reason! It's hard to remember tag names that I've created in the past, also there can be typos when typing them manually.

Create/Edit bookmark... There should be a grid with all existing tags, where we can click/unclick the tags that we want to add/remove to this bookmark. The only reason to type a tag manually is if we are creating a new tag.

Please. Thanks.
